src/Confluent.Kafka/Consumer.cs

```diff
@@ -414,6 +414,28 @@ public void Unassign()
             => consumer.Unassign();
 
         /// <summary>
+        ///     Store offsets for one or more partitions.
+        ///     
+        ///     The offset will be committed (written) to the offset store according
+        ///     to `auto.commit.interval.ms` or manual offset-less commit().
+        /// </summary>
+        /// <remarks>
+        ///     `enable.auto.offset.store` must be set to "false" when using this API.
```

Comment:
What happens if `enable.auto.offset.store` is `true`? We probably want an exception, and to get that we may need to pull out this config property in the Consumer constructor and and check it down on line 1100.